Packs/AWS-IAM/Integrations/AWS-IAM/AWS-IAM.py

```diff
@@ -1009,6 +1009,52 @@ def get_role_policy(args, aws_client):  # pragma: no cover
     return_outputs(human_readable, outputs, response)
 
 
+def get_user_policy(args, aws_client):  # pragma: no cover
+    client = aws_client.aws_session(
+        service=SERVICE,
+        role_arn=args.get('roleArn'),
+        role_session_name=args.get('roleSessionName'),
+        role_session_duration=args.get('roleSessionDuration'),
+    )
+    kwargs = {
+        'UserName': args.get('userName'),
+        'PolicyName': args.get('policyName')
+    }
+    response = client.get_user_policy(**kwargs)
+    response_string = json.dumps(response, default=datetime_to_string)
+    response = json.loads(response_string)
+    outputs = {
+        'AWS.IAM.Users(val.UserName && val.UserName === obj.UserName)':
+            response}
+    del response['ResponseMetadata']
+    table_header = 'AWS IAM User Policy for {}'.format(args.get('userName'))
+
+    return_outputs(response_string, outputs, response)
```

Comment:
These are the best practices for defining the command functions.
And the examples you built on (which appear in this integration)
are no longer updated
You can see [here](https://xsoar.pan.dev/docs/integrations/code-conventions#command-functions) and update according to the updated best practices
mainly to replace the  `return_outputs` with the` return_results`
I am available for any questions and help you may need